File: II semestr/java/laboratorium/instrukcje sterujace/Nwd.java
package pakietzad1a;

import java.util.Scanner;

public class Nwd {

    public static void main(String[] args){
        int x, y;
        Scanner sc = new Scanner(System.in);
        System.out.print("Podaj x: ");
        x = sc.nextInt();
        System.out.print("Podaj y: ");
        y = sc.nextInt();

        while (x != y) {
            if (x > y) {
                x = x - y;
            } else {
                y = y - x;
            }
        }
        System.out.println("Najwiekszy wspolny dzielnik: " + x);
        sc.close();
    }
    }

File: I semestr/C/cwiczenia/nwd.c
//NWD
#include <stdio.h>

int main()
{
    int a,b;

    printf("Podaj pierwsza liczbe: ");
    scanf("%d",&a);

    printf("Podaj podaj druga liczbe: ");
    scanf("%d",&b);

    while(a!=b){
        if(a>b){
            a = a - b;
        }
        else{
            b = b - a;
        }
    }

    printf("Najwieksz wspolny dzielnik to: %d ", a);

    return 0;
}
